CHANGELOG.rst

```diff
@@ -1,5 +1,29 @@
 .. _changelog:
 
+0.12.3
+------
+
+Minor release coming with:
+* several new features on renku-ui,
```

Comment:
```suggestion
Minor release coming with:

* several new features on renku-ui,
```

I believe you need a space here. Otherwise, the formatting is wrong (see preview)
![Screenshot_20220318_143833](https://user-images.githubusercontent.com/43481553/159013140-f9ab63db-608c-4846-b986-0d46ed37e8b4.png)